packages/components/pagination/src/pagination.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import PageNavigator from './page-navigator';
+import PageSizeSelector from './page-size-selector';
+import Spacings from '@commercetools-uikit/spacings';
+
+function Pagination(props) {
+  const totalPages = Math.ceil(props.totalItems / props.pageSize);
+
+  const currentPageItems =
+    props.currentPage === totalPages
+      ? props.totalItems - props.pageSize * (props.currentPage - 1)
+      : props.pageSize;
+
+  return (
+    <Spacings.Inline justifyContent="space-between">
+      <PageSizeSelector
+        options={props.pageSizeOptions}
+        pageSize={props.pageSize}
+        currentPageItems={currentPageItems}
+        onPageSizeChange={props.onPageSizeChange}
+      />
+      <PageNavigator
+        totalPages={totalPages}
+        currentPage={props.currentPage}
+        onPageChange={props.onPageChange}
+      />
+    </Spacings.Inline>
+  );
+}
+Pagination.displayName = 'Pagination';
+Pagination.propTypes = {
+  /**
+   * Number of items per page
+   */
+  pageSize: PropTypes.number.isRequired,
+  /**
+   * Total number of items across all pages
+   */
+  totalItems: PropTypes.number.isRequired,
+  /**
+   * The currently selected page
+   */
+  currentPage: PropTypes.number.isRequired,
+  /**
+   * A callback function, called when the page is changed.
+   * <br>
+   * Signature: `(page: number) => void`
+   */
+  onPageChange: PropTypes.func.isRequired,
+  /**
+   * Options to display as pageSizes
+   */
+  pageSizeOptions: PropTypes.arrayOf(PropTypes.number).isRequired,
```

Comment:
@mariabarrena I don't we remember what we decided here. Do we provide specific options of can you pass whatever value you want?

Reply:
All properties needed are specified in the checklist: https://wiki.commercetools.com/display/shield/Pagination%3A+component+checklist

hasLimit (Boolean):
 if yes, the limit section is displayed. Default is "yes"

LimitValues:
Allows consumers to choose the type of limit between:
• option 20/50: when querying many items is expensive (with possibility to have a third breakpoint of 100 items). Default: 20
• option 200/500: when querying many items does not harm the performance of your application/feature. Default: 200

Reply:
Hmm that doesn't really work.

If we go for pre-defined ranges, maybe we should provide a different prop:

```js
pageSizeRange: 'small' | 'large'
```

Where `small` maps to `20,50,100` and `large` maps to `200,500`